framework/Optimizers/SPSA.py

```diff
@@ -119,7 +119,7 @@ def _newOptPointAdd(self, gradient, traj):
     """
     ak = self._computeGainSequenceAk(self.paramDict,self.counter['varsUpdate'][traj],traj) # Compute the new ak
     self.optVarsHist[traj][self.counter['varsUpdate'][traj]] = {}
-    varK = copy.deepcopy(self.counter['recentOptHist'][traj][0]['inputs'])
+    varK = dict((var,self.counter['recentOptHist'][traj][0][var]) for var in self.getOptVars(traj)) #copy.deepcopy(self.counter['recentOptHist'][traj][0]['inputs'])
```

Comment:
rm commented part?

Reply:
I want to leave this part there just in case the deepcopy ends up coming up again, so I can find it easily.

Reply:
ok